spec/components/tailwinds/form/builder_spec.rb

```diff
@@ -7,13 +7,32 @@
   let(:builder) { Tailwinds::Form::Builder.new :user, resource, view, {} }
 
   describe '#text_field' do
-    let(:output) do
-      builder.text_field :email
+    context 'default' do
+      let(:output) { builder.text_field :email }
+
+      it do
```

Comment:
```suggestion
      it doit 'gets default value' do
```

Reply:
Changed :slightly_smiling_face:

I have not accepted your suggestion because it contains typos :slightly_smiling_face: